docs/Manuals/Program/freeciv21-modpack.rst

```diff
@@ -0,0 +1,54 @@
+..  SPDX-License-Identifier: GPL-3.0-or-later
+..  SPDX-FileCopyrightText: James Robertson <jwrober@gmail.com>
+
+freeciv21-modpack
+*****************
+
+SYNOPSIS
+========
+
+``freeciv21-modpack`` [ -i|--install `<URL>` ] [ -L|--List `<URL>`] [ -h|--help ] [ -v|--version ]
+
+DESCRIPTION
+===========
+
+.. include:: freeciv21-server.rst
+  :start-line: 17
+  :end-line: 29
+
+This is a program to read and install a modpack URL from the command line. If you are looking for a GUI
+based modpack installer, refer to freeciv21-modpack-qt(6).
+
+OPTIONS
+=======
+
+The following options are accepted on the command line of the manual generator. They may not be combined as
```

Comment:
Same here, "manual generator" should be "modpack installer" or something like that?